developer-workflow/c-api.rst

```diff
@@ -29,13 +31,18 @@ for building CPython itself, as indicated by a macro like ``Py_BUILD_CORE``.
 
 While internal API can be changed at any time, it's still good to keep it
 stable: other API or other CPython developers may depend on it.
+For users, internal API is sometimes the best a workaround for a thorny problem
```

Comment:
Like many instances in this doc, "API" is missing an article here, but it's not totally clear which it should be, given the existing semantics not clear (clarifying which being one reason the article is needed in the first place).

If you mean the internal API as a whole here, this should be

```suggestion
For users, the internal API is sometimes the best a workaround for a thorny problem
```

Or if you mean a particular internal API, this should be

```suggestion
For users, internal API is sometimes the best a workaround for a thorny problem
```

Reply:
There's also a typo here "the best a workaround".

Reply:
Thanks, edited to fix above